include/deal.II/lac/vector.templates.h

```diff
@@ -2048,8 +2045,12 @@ Vector<Number>::allocate()
 {
   // make sure that we don't create a memory leak
   Assert (val == 0, ExcInternalError());
-  val = static_cast<Number *>(_mm_malloc (sizeof(Number)*max_vec_size, 64));
-  Assert (val != 0, ExcOutOfMemory());
+
+  // then allocate memory with the proper alignment requirements of 64 bytes
+  const int ierr = posix_memalign ((void **)&val, 64, sizeof(Number)*max_vec_size);
+
```

Comment:
So are you saying that we don't *need* the extra alignment in all of these places, but that it would be nice to have it?